openedxstats/apps/slackdata/apps.py

```diff
@@ -1,6 +1,8 @@
+import django
 from django.apps import AppConfig
 
 
 class SlackdataConfig(AppConfig):
     name = 'slackdata'
-
+    if django.VERSION >= (3, 2):
+        default_app_config = False
```

Comment:
https://docs.djangoproject.com/en/3.2/ref/applications/#django.apps.AppConfig.default

prevent Django from selecting a configuration class automatically.
it already has defined path in Installed apps.